src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -553,6 +550,9 @@ void remove_virtual_function(
 
   rvf.remove_virtual_function(
     goto_program, instruction, dispatch_table, fallback_action);
+
+  remove_skip(goto_program);
```

Comment:
I don't think the singular version should do this -- it's supposed to be lowering a single call into a dispatch table, which should not imply changing unrelated parts of the goto program.

Reply:
Point taken, it might be necessary to come up with a version of `remove_skip` that just takes a range of instructions.

Reply:
I think it would suffice to skip doing this in this case -- the convention is that a *pass* should leave the location numbers and GOTO program in a "tidy" state, and this particular flavour of RVF is not a pass, it's a utility for other passes.

Reply:
Thanks, I'll do a bit more code reading and will then likely just put a comment in there.